assets/src/scripts/components/dailyValueHydrograph/time-series-graph.js

```diff
@@ -59,6 +59,9 @@ const drawDataLines = function (elem, {lines, xScale, yScale, layout}) {
         .attr('y', layout.margin.top)
         .attr('width', layout.width - layout.margin.right)
         .attr('height', layout.height - layout.margin.bottom);
+     if (enableClip) {
```

Comment:
This is using the graph-clip that was created for the hydrograph. We should keep these components independent. If you need a clip rectangle for the DV graph, you should create one in this module, give it a different id (id's should be unique on a page) and then use that. Do you need it? It doesn't look like you are using it for the DV graph itself. Do you need it for both the main DV graph and the brush? If not remove it, otherwise you should create a clip rectangle for the DV component. I'm a little surprised that this works at all since you have set enableClip for the brush and this rectangle is much taller than what you need there.

Reply:
Thanks for catching that, seems to be good now.

Reply:
I wasn't clear. There isn't a dv-graph-clip rectangle to perform this. I see that the enableClip is not getting set for in drawTimeSeriesGraph when drawing the lines. Do you even need this? What happens if you set enableClip to false in the dv graph-brush.j?

Reply:
Oh strange, if I set it to false, they still seem to be clipped.  I swear all the lines were going outside their bounds before this clip was added...hmm